src/current/cockroachcloud/resource-usage-basic.md

```diff
@@ -64,24 +66,27 @@ If any queries are more expensive than expected, you can use the [`EXPLAIN ANALY
+
+1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
+1. Click on the title of the **Execution Count** column to sort your queries by the number of executions of the statements within the time interval.
+1. Sort the queries by the **Rows Processed** or **Bytes Read** column. For most queries, total rows processed should be no more than a few hundred. Most queries should read no more than a few kilobytes per row.
+
 ### Excessive number of connections
 
 CockroachDB {{ site.data.products.basic }} clusters consume minimal resources per connection, so increased RU consumption is not likely to be caused by a high number of connections. However, it will be important to manage your connections for both performance optimization and RU consumption as your application scales up.
 
-Maintaining fewer than five active connections is recommended for most workloads. To diagnose excessive connections, navigate to your cluster's **Metrics** page, [**Monitor SQL Activity** tab]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) in the {{ site.data.products.cloud }} Console. The **SQL Connection Attempts** chart displays new SQL connection attempts over time.
+Maintaining fewer than five active connections is recommended for most workloads. To diagnose excessive connections, navigate to your cluster's [**Overview** metrics page]({% link cockroachcloud/metrics-overview.md %}) in the {{ site.data.products.cloud }} Console. The [**SQL Connections** chart]({% link cockroachcloud/metrics-overview.md %}#sql-connections) displays new SQL connection attempts over time.
 
 [Connection pooling]({% link {{site.current_cloud_version}}/connection-pooling.md %}) is the recommended way to manage the number of connections for many workloads. To read more about connection pooling, see [What is Connection Pooling, and Why Should You Care](https://www.cockroachlabs.com/blog/what-is-connection-pooling/).
 
 ### Excessive data egress
 
-In a CockroachDB {{ site.data.products.basic }} deployment, the {{ site.data.products.cloud }} Console does not provide direct observability of data egress, but you can observe the component of egress that comes from SQL statements:
-
-1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
-1. Sort the queries by the **Bytes Read** column. Most queries should read fewer than 1000 bytes.
+To diagnose excessive data egress, navigate to your cluster's [**Request Units** metrics page]({% link cockroachcloud/metrics-request-units.md %}) in the {{ site.data.products.cloud }} Console and monitor the [**Egress** chart]({% link cockroachcloud/metrics-request-units.md %}#egress) for high RU's for client traffic or bulk I/O operations.
```

Comment:
I'd change all instances of `RU's` to "RUs". (There are several in this doc.)

Reply:
good catch. updated